README.md

```diff
@@ -66,6 +66,18 @@ Per-VDOM:
  * `fortigate_link_bandwidth_rx_byte_per_second`
  * `fortigate_link_status_change_time_seconds`
 
+ Per-SDWAN and VDOM:
+ * `fortigate_virtual_wan_healthcheck_status`
+ * `fortigate_virtual_wan_healthcheck_latency_seconds`
+ * `fortigate_virtual_wan_healthcheck_jitter_seconds`
+ * `fortigate_virtual_wan_healthcheck_packetloss_ratio`
```

Comment:
packet_loss / packet_sent / etc to match link-monitor metrics.

Reply:
Is it a problem if the metrics are named the same ? Even if the subcommand "virtual_wan" is different form "link"?

Reply:
Consistency is better, don't you agree?
It is a bit annoying to see one packetloss and one packet_loss - I think we should stick to one style instead of introducing another one unless there is a clear win.